fido/prepare.py

```diff
@@ -190,13 +207,13 @@ def parse_pronom_xml(self, source, puid_filter=None):
                 bytes = get_text_tna(pronom_pat, 'ByteSequenceValue')
                 offset = get_text_tna(pronom_pat, 'Offset')
                 max_offset = get_text_tna(pronom_pat, 'MaxOffset')
-                if max_offset == None:
+                if not max_offset:
                     pass
-                #print "working on puid:", puid, ", position: ", pos, "with offset, maxoffset: ", offset, ",", max_offset
+                # print "working on puid:", puid, ", position: ", pos, "with offset, maxoffset: ", offset, ",", max_offset
                 regex = convert_to_regex(bytes, 'Little', pos, offset, max_offset)
-                #print "done puid", puid
+                # print "done puid", puid
                 if regex == "__INCOMPATIBLE_SIG__":
-                    print >> sys.stderr, "Error: incompatible PRONOM signature found for puid", puid, ", skipping..."
+                    print("Error: incompatible PRONOM signature found for puid {} skipping...".format(puid), file=sys.stderr)
```

Comment:
I know, I think I tend to like single strings because I've done so much i18n where I would probably be doing something like the following:

````python
print(_('Error: incompatible PRONOM signature found for puid %s skipping...') % puid)
```